Analysis/Core/include/AnalysisCore/HFSelectorCuts.h

```diff
@@ -0,0 +1,86 @@
+                                               {6.0, 6.5, 0.400, 300. * 1E-4, 0.8, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, -8000. * 1E-8, 0.85, 0., 0.},      /* 6<pt<6.5 */
+                                               {6.5, 7.0, 0.400, 300. * 1E-4, 0.8, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, -8000. * 1E-8, 0.85, 0., 0.},      /* 6.5<pt<7 */
+                                               {7.0, 7.5, 0.400, 300. * 1E-4, 0.8, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, -7000. * 1E-8, 0.85, 0., 0.},      /* 7<pt<7.5 */
+                                               {7.5, 8.0, 0.400, 300. * 1E-4, 0.8, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, -7000. * 1E-8, 0.85, 0., 0.},      /* 7.5<pt<8 */
+                                               {8.0, 9.0, 0.400, 300. * 1E-4, 0.9, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, -5000. * 1E-8, 0.85, 0., 0.},      /* 8<pt<9 */
+                                               {9.0, 10.0, 0.400, 300. * 1E-4, 0.9, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, -5000. * 1E-8, 0.85, 0., 0.},     /* 9<pt<10 */
+                                               {10.0, 12.0, 0.400, 300. * 1E-4, 0.9, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, -5000. * 1E-8, 0.85, 0., 0.},    /* 10<pt<12 */
+                                               {12.0, 16.0, 0.400, 300. * 1E-4, 1.0, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, 10000. * 1E-8, 0.85, 0., 0.},    /* 12<pt<16 */
+                                               {16.0, 20.0, 0.400, 300. * 1E-4, 1.0, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, 999999. * 1E-8, 0.85, 0., 0.},   /* 16<pt<20 */
+                                               {20.0, 24.0, 0.400, 300. * 1E-4, 1.0, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, 999999. * 1E-8, 0.85, 0., 0.},   /* 20<pt<24 */
+                                               {24.0, 36.0, 0.400, 300. * 1E-4, 1.0, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, 999999. * 1E-8, 0.85, 0., 0.},   /* 24<pt<36 */
+                                               {36.0, 50.0, 0.400, 300. * 1E-4, 1.0, 0.7, 0.7, 1000. * 1E-4, 1000. * 1E-4, 999999. * 1E-8, 0.85, 0., 0.},   /* 36<pt<50 */
+                                               {50.0, 100.0, 0.400, 300. * 1E-4, 1.0, 0.6, 0.6, 1000. * 1E-4, 1000. * 1E-4, 999999. * 1E-8, 0.80, 0., 0.}}; /* pt>50 */
+
+// row labels
+static const std::vector<std::string> pTBinLabels = {
+  "pT bin 0",
+  "pT bin 1",
+  "pT bin 2",
+  "pT bin 3",
+  "pT bin 4",
+  "pT bin 5",
+  "pT bin 6",
+  "pT bin 7",
+  "pT bin 8",
+  "pT bin 9",
+  "pT bin 10",
+  "pT bin 11",
+  "pT bin 12",
+  "pT bin 13",
+  "pT bin 14",
+  "pT bin 15",
+  "pT bin 16",
+  "pT bin 17",
+  "pT bin 18",
+  "pT bin 19",
+  "pT bin 20",
+  "pT bin 21",
+  "pT bin 22",
+  "pT bin 23",
+  "pT bin 24"};
+
+// column labels
+static const std::vector<std::string> cutVarLabels = {"pT lo", "pT hi", "m", "DCA", "cos theta*", "pT K", "pT pi", "d0K", "d0pi", "d0d0", "cos pointing angle", "cos pointing angle xy", "normalized decay length XY"};
```

Comment:
Here maybe we can change some names:
pT low
pT High
Pi
for the longer names at the end it might be easier to find a bit of a short hand, but i agree it also makes it clear